### PR TITLE
Lighthouse suggestions

### PR DESCRIPTION
**Pull request by martijnb92**
Some minor adjustments suggested by [Lighthouse](https://developers.google.com/web/tools/lighthouse), all scores should be 90+ now.

### CODE REVIEW

**Comment by pxpm** [comment]
I am all-in for `rel="noopener"` 

Nice catch @martijnb92

**Comment by tabacitu** [comment]
Totally! Thank you @martijnb92 & @pxpm !

File: src/resources/views/base/inc/main_header.blade.php [hunk 1]
```diff
@@ -1,12 +1,12 @@
 <header class="{{ config('backpack.base.header_class') }}">
   <!-- Logo -->
-  <button class="navbar-toggler sidebar-toggler d-lg-none mr-auto ml-3" type="button" data-toggle="sidebar-show">
+  <button class="navbar-toggler sidebar-toggler d-lg-none mr-auto ml-3" type="button" data-toggle="sidebar-show" aria-label="{{ trans('backpack::base.toggle_navigation')}}">
     <span class="navbar-toggler-icon"></span>
   </button>
-  <a class="navbar-brand" href="{{ url(config('backpack.base.home_link')) }}">
+  <a class="navbar-brand" href="{{ url(config('backpack.base.home_link')) }}" title="{{ config('backpack.base.project_name') }}">
```
Comment (pxpm) [inline src/resources/views/base/inc/main_header.blade.php:6]: Looking at it, why the change from `home_link` to `project_name` ? @martijnb92

Comment (martijnb92) [reply src/resources/views/base/inc/main_header.blade.php:6]: `home_link` is not changed, only a title attribute with the project name is added.

Comment (pxpm) [reply src/resources/views/base/inc/main_header.blade.php:6]: Sorry you are right. Read it like :upside_down_face:
